app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1> New page </h1>
+
+<%= form_with model: @task, class: 'task-form' do |f| %>
+<p> Please provide a new task: </p>
+  <div>
+    <%= f.label :name %>:
+    <%= f.text_field :name %>
+  </div>
+  <div>
+    <%= f.label :description %>:
+    <%= f.text_field :description %>
+  </div>
+  <div>
+    <%= f.label :completed_at %>:
+    <%= f.text_field :completed_at %>
+  </div>
```

Comment:
Consider whether `completed_at` should be submitted through a form. Consider initializing a new task with `nil` for completed_at.